<launch>

  <arg name="recordData" default="false" />

  <node pkg="rl_pkg" name="hand_control" type="hand_control.py" output="screen">
        <rosparam command="load" file="$(find rl_pkg)/param/control.yaml" />
  </node>
  <node pkg="rl_pkg" name="runPG" type="runPG.py" output="screen"/>

  <node pkg="openhand" name="openhandNode" type="openhandNode.py" output="screen">
		<rosparam command="load" file="$(find dexterous_manipulation)/param/model_t42.yaml"/>
  </node>

  <node name="camera" pkg="usb_cam" type="usb_cam_node">
	<param name="image_width" value="1024"/>
	<param name="image_height" value="576"/>
  </node>

 <node pkg="rosbag" type="record" name="rosbag_record_data"
       args="record -O /home/pracsys/Dropbox/transfer/F /gripper_t42/vel_ref_monitor /gripper_t42/pos_ref_monitor /marker_tracker/image_space_pose_msg /gripper/load /gripper/pos /gripper/curr /keyboard_input"
       if="$(arg recordData)" />

  <node pkg="dexterous_manipulation" type="plot_obj_pos.py" name="Plot" output="screen"></node>
  <node name="marker_tracker" pkg="marker_tracker" type="marker_tracker"/>
  <node name="gripper_state_publisher" pkg="gripper_state_publisher" type="gripper_state_publisher_node"/>

</launch>